code_map.py

```diff
@@ -325,16 +387,37 @@ def get_mapper(file):
             if file.lower().endswith('.py'):
                 return Mapper.python_mapper.generate, py_syntax
 
-    def can_map(file):
-        if path.basename(file) == "Code - Map":
-            return None
-        return code_map_generator.get_mapper(file) is not None
+    # -----------------
+
+    def view_to_map(view):
+
+        file_syntax = view.settings().get('syntax')
+        supported = Mapper.settings().get('syntaxes')
+        supported.remove(['universal', ""])
+
+        matched_syntax = False
+        for syntax in supported:
+            if syntax[0] in file_syntax.lower():
+                matched_syntax = True
+                break
+
+        if not matched_syntax:
+            return ("Could not decode view.", txt_syntax)
+
+        content = view.substr(Region(0, view.size()))
+        mapper = Mapper.universal_mapper.generate(content)
+        if mapper:
+            TEMP_VIEWS[view.id()] = view
+            return (mapper, file_syntax)
+        else:
+            return ("Could not decode view.", txt_syntax)
 
```

Comment:
This is how it takes the view content and stores it. For now it will only work with universal mapper, and only if a syntax is matched.